src/atomate2/common/schemas/phonons.py

```diff
@@ -84,16 +93,16 @@ def __init__(self, force_constants: List[List[Matrix3D]]):
 class PhononJobDirs(BaseModel):
     """Collection to save all job directories relevant for the phonon run."""
 
-    displacements_job_dirs: List[Optional[str]] = Field(
+    displacements_job_dirs: List[str] = Field(
         None, description="The directories where the displacement jobs were run."
     )
-    static_run_job_dir: Optional[str] = Field(
+    static_run_job_dir: str = Field(
         None, description="Directory where static run was performed."
     )
-    born_run_job_dir: Optional[str] = Field(
+    born_run_job_dir: str = Field(
         None, description="Directory where born run was performed."
     )
-    optimization_run_job_dir: Optional[str] = Field(
+    optimization_run_job_dir: str = Field(
```

Comment:
There were some recent updates. Maybe make sure this is included

Reply:
made this change will do a diff check once the forcefields are adapted

Reply:
I ran a diff on all of the common phonon files and double checked the VASP ones I think everything is consistent, but I will do another check in the morning to be sure

Reply:
Everything should be consistent now